Controller/CRUDController.php

```diff
@@ -748,6 +734,40 @@ public function createAction(Request $request = null)
     }
 
     /**
+     * Add flash message when success create new object in createAction
```

Comment:
This description is not really appropriate because this method could be used elsewhere instead of createAction.